src/directives/class-map.ts

```diff
@@ -66,55 +48,44 @@ export const classMap = directive((classInfo: ClassInfo) => (part: Part) => {
 
   let previousClasses = previousClassesCache.get(part);
   if (previousClasses === undefined) {
-    previousClasses = new Map();
-    previousClassesCache.set(part, previousClasses);
-    // Normalize all static classes into individual tokens. This is necessary
-    // since each individual string could contain multiple tokens.
-    const strings = committer.strings.join(' ');
-    let match;
-    while ((match = tokens.exec(strings))) {
-      // Ensure static classes are never removed, by setting them to true
-      previousClasses!.set(match[0], true);
-    }
-    changed = true;
+    // Write static classes once
+    // Use setAttribute() because className isn't a string on SVG elements
+    element.setAttribute('class', committer.strings.join(' '));
+    previousClassesCache.set(part, previousClasses = new Set());
   }
 
+  // IE11 doesn't support classList on SVG elements, so we emulate it with a Set
+  const classes = new Set((element.getAttribute('class') || '').split(/\s+/));
```

Comment:
We had a bug before about `Set` not supporting iterable in the constructor.

Reply:
I remember that too, but checked MDN and tried it... maybe that's the Chrome 41 error too.